pkg/auth/rbac.go

```diff
@@ -436,20 +463,22 @@ func internalMatchNamespace(srcNs string, namespaces []*security.StringMatch) bo
 
 func (r *Rbac) buildConnV4(tupleV4 *bpfSockTupleV4) rbacConnection {
 	conn := rbacConnection{}
-	conn.srcIp = binary.LittleEndian.AppendUint32(conn.srcIp, tupleV4.SrcAddr)
-	conn.dstIp = binary.LittleEndian.AppendUint32(conn.dstIp, tupleV4.DstAddr)
-	conn.dstPort = uint32(tupleV4.DstPort<<8 | tupleV4.DstPort>>8)
+	// srcIp and dstIp are big endian, and dstPort is little endian, which is consistent with authorization policy flushed to Kmesh
+	conn.srcIp = binary.BigEndian.AppendUint32(conn.srcIp, tupleV4.SrcAddr)
```

Comment:
why not make srcIp  as net.IP type.

Reply:
Merge `binary.Read` and `buildConnV4` into one function